lib/home/view/tasks_page.dart

```diff
@@ -12,7 +13,10 @@ class TasksPage extends StatelessWidget {
       appBar: AppBar(
         title: const Text('To Do List'),
       ),
-      body: const TaskListItem(),
+      body: Center(
+        child: TaskCard( task: Task('Lers Flutter', true),
```

Comment:
TaskCard takes Task instance.
for development purposes you can create Task instance in widget as you did here, but much nicer way is to take it from https://github.com/kamilgadek/to_do_list_kamilg/blob/main/packages/tasks_repository/lib/src/data.dart